Loop/Managers/LoopDataManager.swift

```diff
@@ -99,8 +99,8 @@ final class LoopDataManager: ObservableObject {
     }
 
     @Published private(set) var lastLoopCompleted: Date?
-    @Published private(set) var publishedMostRecentGlucoseDataDate: Date?
-    @Published private(set) var publishedMostRecentPumpDataDate: Date?
+    @Published private(set) var publishedMostRecentGlucoseDataDate: Date? = nil
```

Comment:
I don't think you need the assignments here, as they get assigned below?

Reply:
Since they're being assigned by computed properties (before self is fully initialized) they either need to be set to `nil` here or I can move the `nil` assignment to the initializer before assigning the computed value to them.

Reply:
Good call. Since they're of an optional `Date` type `nil` should be the default anyway.